po/fr.po

```diff
@@ -9032,6 +9035,342 @@ msgstr "N/B Kodak T-max 400"
+"ne peut être définie donc ce que vous avez à faire est d'utiliser un "
+"illuminant personnalisé."
+
+#. Parameters did not change, assume user wants to edit auto-set params and display controls
+#: ../src/iop/channelmixerrgb.c:1530
+msgid "white balance successfuly extracted from raw image"
+msgstr "la balance couleur a été extraite de l'image raw avec succèss"
+
+#: ../src/iop/channelmixerrgb.c:1548
+msgid "no white balance was found in raw image"
+msgstr "aucune balance couleur n'a été trouvée dans l'image raw"
+
+#. We need to recompute only the thumbnail
+#: ../src/iop/channelmixerrgb.c:1559
+msgid "auto-detection of white balance started…"
+msgstr "l'auto-détection de la balance couleur a démarrée..."
+
+#: ../src/iop/channelmixerrgb.c:2198
+msgid "CAT"
+msgstr "CAT"
+
+#: ../src/iop/channelmixerrgb.c:2202
+msgid "colorfulness"
+msgstr "colorimétrie"
+
+#: ../src/iop/channelmixerrgb.c:2204
+msgid "grey"
+msgstr "gris"
+
+#: ../src/iop/channelmixerrgb.c:2209
+msgid "adaptation"
+msgstr "adaptation"
+
+#: ../src/iop/channelmixerrgb.c:2210
+msgid "linear Bradford (ICC v4)"
+msgstr "Bradford linéaire (ICC v4)"
+
+#: ../src/iop/channelmixerrgb.c:2211
+msgid "CAT16 (CIECAM16)"
+msgstr "CAT16 (CIECAM16)"
+
+#: ../src/iop/channelmixerrgb.c:2212
+msgid "non-linear Bradford"
+msgstr "Bradford non-linéaire"
+
+#: ../src/iop/channelmixerrgb.c:2213
+msgid "XYZ "
+msgstr "XYZ"
+
+#: ../src/iop/channelmixerrgb.c:2214
+msgid "none (bypass)"
+msgstr "aucun (contourner)"
+
+#: ../src/iop/channelmixerrgb.c:2215
+msgid ""
+"choose the method to adapt the illuminant\n"
+"and the colorspace in which the module works: \n"
+"• Linear Bradford (1985) is more accurate for illuminants close to daylight\n"
+"but produces out-of-gamut colors for difficult illuminants.\n"
+"• CAT16 (2016) is more robust to avoid imaginary colours\n"
+"while working with large gamut or saturated cyan and purple.\n"
+"• Non-linear Bradford (1985) is the original Bradford,\n"
+"it can produce better results than the linear version, but is unreliable.\n"
+"• XYZ is a simple scaling in XYZ space. It is not recommended in general.\n"
+"• none disables any adaptation and uses pipeline working RGB."
+msgstr ""
+"choisir la méthode pour ajuster l'illuminant\n"
+"et l'espace couleur dans lequel le module travaille :\n"
+"• Bradford linéaire (1985) est plus approprié pour les illuminants proche de "
+"la lumière du jour\n"
+"mais produit des couleurs hors gamut pour les illuminants difficiles\n"
+"• CAT16 (2016) est plus robuste pour éviter les couleurs imaginaires\n"
+"tout en travaillant avec un large gamut ou du violet et cyan saturés.\n"
+"• Bradford non-linéaire (1985) est le Bradford originel,\n"
+"il peut produire de meilleurs résultats que la version linéaire, mais est "
+"peu fiable.\n"
+"• XYZ est une simple mise à l'échelle de l'espace XYZ. Il n'est en général "
+"pas recommandé.\n"
+"• aucun désactive toute adaptation et utilise la chaîne de traitement RVB de "
```

Comment:
utilise l'espace RGB de travail du pipeline

Reply:
*RVB in french